.github/workflows/staging.yml

```diff
@@ -59,8 +59,16 @@ jobs:
         directory: ./uberspace/
         key: ${{secrets.STAGING_SSH_PRIVATE_KEY}}
         inventory: |
-          [servers]
-          despina.uberspace.de ansible_user=staghios domain=staging.ephios.de smtp_mailbox=noreply admin_email="Ephios Admins <admin@ephios.de>"
+          servers:
+            hosts:
+              staghios:
```

Comment:
`staghios` is an "inventory alias": https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#inventory-aliases